session/session.go

```diff
@@ -95,6 +95,16 @@ var (
 	sessionExecuteParseDurationGeneral    = metrics.SessionExecuteParseDuration.WithLabelValues(metrics.LblGeneral)
 )
 
+var gcVariableComments = map[string]string{
+	variable.TiKVGCRunInterval:  "GC run interval, at least 10m, in Go format.",
+	variable.TiKVGCLifetime:     "All versions within life time will not be collected by GC, at least 10m, in Go format.",
+	variable.TiKVGCConcurrency:  "How many goroutines used to do GC parallel, [1, 128], default 2",
+	variable.TiKVGCEnable:       "Current GC enable status",
+	variable.TiKVGCMode:         "Mode of GC, \"central\" or \"distributed\"",
+	"tikv_gc_auto_concurrency":  "Let TiDB pick the concurrency automatically. If set false, tikv_gc_concurrency will be used",
```

Comment:
I can make it a constant in session. It is not present in the variable package, because `tikv_gc_auto_concurrency` is set via `TiKVGCConcurrency` (MySQL uses `-1` to mean "auto").